src/test/java/com/microsoft/azure/relay/SendReceiveTest.java

```diff
@@ -75,29 +76,33 @@ public void createClient() throws URISyntaxException {
 	@Test
 	public void websocketSmallSendSmallResponseTest() {
 		CompletableFuture<Void> listenerTask = websocketListener(smallStr, smallStr);
-		websocketClient(smallStr, smallStr);
+		CompletableFuture<Void> clientTask = websocketClient(smallStr, smallStr);
 		listenerTask.join();
+		clientTask.join();
 	}
 	
 	@Test
 	public void websocketSmallSendLargeResponseTest() {
 		CompletableFuture<Void> listenerTask = websocketListener(smallStr, largeStr);
-		websocketClient(largeStr, smallStr);
+		CompletableFuture<Void> clientTask = websocketClient(largeStr, smallStr);
 		listenerTask.join();
+		clientTask.join();
 	}
 	
 	@Test
 	public void websocketLargeSendSmallResponseTest() {
 		CompletableFuture<Void> listenerTask = websocketListener(largeStr, smallStr);
-		websocketClient(smallStr, largeStr);
+		CompletableFuture<Void> clientTask = websocketClient(smallStr, largeStr);
```

Comment:
>websocketClient [](start = 39, length = 15)

nothing to do in this PR, I just want to point out this method name is a noun only and method names are more self-documenting when they contain verbs too.

Reply:
e.g. RunClientTest, TestClient, ProcessClient, ConfigureClient, etc.

---
In reply to: [294378342](https://github.com/Azure/azure-relay-java/pull/47#discussion_r294378342) [](ancestors = 294378342)